docs/library/machine.I2C.rst

```diff
@@ -112,19 +112,21 @@ control over the bus, otherwise the standard methods (see below) can be used.
 
    Availability: ESP8266.
 
-.. method:: I2C.readinto(buf)
+.. method:: I2C.readinto(buf, nack=True)
```

Comment:
@dhylands according to the specs the master sends ACK/NACK when reading from the slave.  Look at UM10204 rev 6 (google it), sec 3.1.6 uses "transmitter" and "receiver" terminology when talking about ACK/NACK (not master/slave).  In particular look at fig 12 and you'll see that the master is sending the ACK and NACK bits when reading from a slave.